src/webgl/camera.js

```diff
@@ -34,20 +46,129 @@ var p5 = require('../core/core');
  * blue square shrinks in size grows to fill canvas. disappears then loops.
  *
  */
-p5.prototype.camera = function(x, y, z){
-  //what it manipulates is the model view matrix
-  this._renderer.translate(-x, -y, -z);
+p5.prototype.camera = function(){
```

Comment:
Can this long section of camera math be simplified using the p5.Vector class? Or other math primitives?

I realize speed might be slightly compromised by not having everything inlined...